src/electron/platform/android/rule-information-provider.ts

```diff
@@ -80,8 +80,9 @@ export class RuleInformationProvider {
 
     private getTouchSizeUnifiedFormattableResolution = (ruleResultsData: RuleResultsData): UnifiedFormattableResolution => {
         const dpi = ruleResultsData.props['Screen Dots Per Inch'] as number;
-        const physicalWidth = ruleResultsData.props['width'] as number;
-        const physicalHeight = ruleResultsData.props['height'] as number;
+        const boundingRect = ruleResultsData.props['boundsInScreen'];
+        const physicalWidth = (boundingRect['right'] as number) - (boundingRect['left'] as number);
```

Comment:
Do we want to make this much more type strict by not casting the `boundingRect['right']` as number and making sure that `physicalWidth` is declared as `const physicalWidth: number`? Seems like that would be a better check.

Reply:
Because `string` to `number` conversions in `javascript` (which TS will eventually compile down to) are so uncertain; I think we should use TS to strictly here to avoid those potential runtime problems.

Reply:
Changed as suggested